recipe/meta.yaml

```diff
@@ -1,45 +1,51 @@
 {% set name = "minimal-snowplow-tracker" %}
 {% set version = "0.0.2" %}
 
-
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/minimal-snowplow-tracker-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: acabf7572db0e7f5cbf6983d495eef54081f71be392330eb3aadb9ccb39daaa4
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.6
+    - python
     - requests >=2.2.1,<3.0
     - six >=1.9.0,<2.0
 
 test:
   imports:
     - snowplow_tracker
     - snowplow_tracker.test
-  commands:
-    - pip check
   requires:
     - pip
+  commands:
+    - pip check
 
 about:
-  home: https://www.fishtownanalytics.com
-  dev_url: https://pypi.org/project/minimal-snowplow-tracker/
+  home: https://www.getdbt.com/
```

Comment:
```suggestion
  home: https://www.getdbt.com/
  dev_url: https://github.com/dbt-labs/snowplow-python-tracker
```

Reply:
No, it's another package with other version numbers. There is no v0.0.2

Reply:
The files you are installing are from that repository, yes: https://github.com/dbt-labs/snowplow-python-tracker/commit/bf630807ab93326ada84044a06a77ce2e9ce7b33

you can compare the date of this commit also with the date of the upload in pypi. also compare the file content

the snowplow-tracker package with newer versions is avaliable somewhere else. the tags are just poorly maintained. you can ignore them

Reply:
But there isn't a tag for v0.0.2. `dbt` requires **==0.0.2** https://github.com/dbt-labs/dbt-core/blob/v1.5.3/core/setup.py#L58

Reply:
a git tag is just a convention, nothing in pip or conda, etc, relies on tags. as long as the maintainer uploaded the source to pypi it works, no matter if tags are correct or not

Reply:
Fixed